geomet_data_registry/layer/base.py

```diff
@@ -160,16 +159,18 @@ def update_count(self, item, r, item_dict):
             LOGGER.debug('Look if we have a complete model run')
             if int(new_layer_file_count) >= item['expected_count']:
                 for mr in self.model_run_list:
-                    layer_count_key_reset = '{}_{}_count'.format(
-                        item_dict['properties']['layer'], mr)
+                    layer_count_key_reset = '{}_{}_{}_count'.format(
+                        self.model, self.wx_variable, mr)
                     self.store.set_key(layer_count_key_reset, 0)
-                    # self.new_key_store = True
+                    self.new_key_store = True
             elif int(new_layer_file_count) == 1:
                 for mr in self.model_run_list:
-                    layer_count_key_reset = '{}_{}_count'.format(
-                        item_dict['properties']['layer'], mr)
-                    mr_nm = int(self.store.get_key(layer_count_key_reset))
-                    if layer_count_key_reset != layer_count_key and mr_nm != 0:
+                    layer_count_key_reset = '{}_{}_{}_count'.format(
+                        self.model, self.wx_variable, mr)
+                    mr_nm = self.store.get_key(layer_count_key_reset)
+                    if layer_count_key_reset != layer_count_key and \
```

Comment:
Better written as:
```python
                    if layer_count_key_reset != layer_count_key and mr_nm not in ['0', None]:
```

Shouldn't `mr_nm` be an int (not `'0'`)?

Reply:
yeah I remember now why I used that: 

we have: 
```
mr_nm = self.store.get_key(layer_count_key_reset)
```
Here when we have a cold start, `mr_nm = None` nad then I can't None to an integer: 
```
mr_nm = int(self.store.get_key(layer_count_key_reset))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

Reply:
still I have updated the if statement like you suggested: `...and mr_nm not in ['0', None]:`